### DISCUSSION TITLE
Dimmer

### DISCUSSION THREAD

**Discussion by Predator6464** [arendst/Tasmota#11523] [Q&A] [answered]
Hi! I am trying to have some dimmable lamps in my house, but I am having some problems.

I bought a dimmer that has this shape:

[https://templates.blakadder.com/qs-wifi_D01_dimmer.html](https://templates.blakadder.com/qs-wifi_D01_dimmer.html)

Although in the description indicated to be an MS-105:

[https://templates.blakadder.com/moes-MS-105-1.html](https://templates.blakadder.com/moes-MS-105-1.html)

I flashed it with Tuya-Convert and I put the command "TuyaMCU 21,2" on it, and it seemed to work fine. I couldn't test it with a dimmable bulb.

Now I have bought a dimmable light bulb, but I do not know if the operation is normal.

Is it normal for the light to take 3-4 seconds to turn on? I have tried commands like Fade and Speed and they do nothing.

From 0% to 40%, the bulb remains off, without emitting any light. I imagine I can fix it with DimmerRange, is it normal?

Testing with one of my dimmers (I have two of the same), I think it has broken. When it goes below 60%, it does not matter what value you put, it no longer reacts. I'll try flashing it again, but it looks bad. Do you recommend a similar dimmer?

Thank you very much, I'm a little desperate.

**Comment by sfromis** [comment, marked as answer]
I do not know specifics there, but with a TuyaMCU being the main controller, Tasmota loses a *lot* of control over device operation. It may be possible to config some details, but that's outside of what I know about.... https://tasmota.github.io/docs/TuyaMCU/#dimmer

**Reply by Predator6464** [reply]
I see, I was imagining something like that. Do you recommend another dimmer? Thanks!

**Reply by sfromis** [reply]
I'm not much into dimmers.... Did you try DimmerRange? It sounds plausible that you could get around the 0/40% darkness using that, which might help with the turn-on time too.
